web/src/utils/date.ts

```diff
@@ -81,3 +81,52 @@ export const getDaysBetween = (startDate: string, endDate: string): DateTime[] =
 
   return dates
 }
+
+export class SmartDate {
```

Comment:
Is this used anywhere? Why is it in this PR?

Reply:
Oh sweet - the references were removed by the merge. I'll kill it.

Reply:
web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx : 120

I have plans for this class.

Reply:
Can you please write tests for it?

Reply:
Or move it to it's own PR if it's only used in one place, this PR is already huge.

Reply:
Pulled out.

Reply:
@Sybrand it's still here but not used?

Reply:
I was just about to blame github when I noticed I hadn't pressed save!

Reply:
Push commit. I blame the bee sting.